sdk/core/Azure.Core/src/OperationOfT.cs

```diff
@@ -11,21 +11,15 @@
 namespace Azure
 {
     /// <summary>
-    /// Represents a long-running operation.
+    /// Represents a long-running operation that returns a value when it completes.
     /// </summary>
     /// <typeparam name="T">The final result of the long-running operation.</typeparam>
 #pragma warning disable SA1649 // File name should match first type name
 #pragma warning disable AZC0012 // Avoid single word type names
-    public abstract class Operation<T> where T : notnull
+    public abstract class Operation<T>: Operation where T : notnull
```

Comment:
Response\<T\> doesn't inherit from Response, do we want / need this inheritance here?

Reply:
The plan here is to allow client libraries to start with operations that don't return a value but still be able to add value in the future by inheriting from `Operation<T>` instead of `Operation `